src/core/Akka.Cluster/ClusterSettings.cs

```diff
@@ -300,6 +301,8 @@ TimeSpan GetWeaklyUpDuration()
         /// The leader will move <see cref="MemberStatus.WeaklyUp"/> members to <see cref="MemberStatus.Up"/> status once convergence has been reached.
         /// </summary>
         public TimeSpan WeaklyUpAfter { get; }
+        
+        public bool UseLegacyHeartbeatMessage { get; }
```

Comment:
Should also add an XML-DOC comment here. Can do that in a future PR.

Reply:
Done